web-app/src/components/SettingsDialog.js

```diff
@@ -478,12 +478,44 @@ class SettingsDialog extends React.Component {
 									style={{
 										color: "rgba(40, 26, 26, 0.56)",
 										marginTop: "16px",
-										padding: "16px 0px",
+										padding: "16px 0px 0px 0px",
 									}}
 								>
-									Custom nation colours
+									Nation colours
+								</Typography>
+								<div width="100%">
```

Comment:
I'm not sure this will work. You should use the styles object as described below :)

Reply:
? I don't get this comment - I added an extra header.

Reply:
`div` doesn't have an attribute called `width`. Needs to either be applied through inline styling (bad!) or using the style object at the top of the script

Reply:
-_- doh